docs/kits.md

```diff
@@ -0,0 +1,139 @@
+# CMake kits
+
+A _kit_ defines project-agnostic and configuration-agnostic info about how to build code. A kit can include:
+
+- A set of compilers: these are locked at specific versions so that you can switch your compiler version quickly and easily.
+- A Visual Studio installation: building for Visual Studio involves more than just finding the necessary compiler executable. Visual C++ requires certain environment variables to be set to tell it how to find and link to the Visual C++ toolchain headers and libraries.
+- A toolchain file: The low-level way to instruct CMake how to compile and link for a target. CMake Tools handles toolchain files using kits.
+
+Kits are mostly CMake-generator-agnostic (a CMake generator writes the input files for the native build system). Visual Studio kits have a preferred generator that will be used as a fallback to ensure a matching MSBuild and .sln generator are used for the Visual C++ compiler.
+
+> **Note:**
+> * If you use the [Ninja](https://ninja-build.org/) build system, don't worry about Visual Studio CMake Generators. CMake Tools will prefer Ninja if it is present, unless configured otherwise.
+> * If you change the active kit while a project is configured, the project configuration will be re-generated with the chosen kit.
+> * Using a kit is recommended but optional. If you don't use a kit, CMake will perform its own automatic detection.
+
+## How kits are found and defined
+
+When the CMake Tools extension runs for the first time, it will [scan for kits](#scan-for-kits) to find available toolchains. It populates the initial list of kits by looking in directories where known compilers are typically installed, and uses `vswhere` to find Visual Studio installations.
+
+### User-local kits
+
+A user-local kit is a kit that is available to a particular user for all projects open with CMake Tools.
+
+The user-local list of kits is stored in the `cmake-kits.json` file, which you can edit by invoking **Edit CMake Kits** from the command palette:
+
+![Example cmake_kits_json file](images/cmake_kits_json.png)
+
+You can manually edit this file to define new global kits. The contents of this file will be automatically updated by CMake Tools during a [scan for kits](#scan-for-kits).
+
+> **Warning:**
+> Don't modify kits that the CMake Tools extension creates because your changes will be overwritten during the next [scan for kits](#scan-for-kits).
```

Comment:
Maybe add something - "You should instead define a new kit with you desired modifications" to be explicit about the preferred behavior

Reply:
I modified this section to read:

You can manually edit this file to define new global kits, however the contents of this file will be automatically updated by CMake Tools during a [scan for kits](#scan-for-kits).

> **Tip:**
> Define a new kit with your desired settings rather than modify kits that the CMake Tools extension creates so that your changes aren't overwritten during the next [scan for kits](#scan-for-kits).